src/Monitor/Monitor/ChangeLog.md

```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
 
+## Version 1.2.2
```

Comment:
This just goes under the 'Upcoming release' heading, and our versioning tools will determien the new version number, per the comments above in the file.